docs/lib/predictions/fragments/ios/identify-entity.md

```diff
@@ -70,3 +74,27 @@ func detectEntities(_ image: URL) {
 	})
 }
 ```
+
+### Detecting Celebrities
+
+To detect celebrities you can pass in `.detectCelebrity` in the `type:` field.  Results are mapped to `IdentifyCelebritiesResult`.  For example:
+
+``` swift
+func detectCelebs(_ image: URL) {
+  _ = Amplify.Predictions.identify(type: .detectCelebrity, image: image, options: PredictionsIdentifyRequest.Options(), listener: { (event) in
+    switch event {
+    case .completed(let result):
+      let data = result as! IdentifyCelebritiesResult
+      if let detectedCeleb = data.celebrities.first {
+        print("Celebrity Name: \(detectedCeleb.metadata.name)")
+      }
+      print(result)
+    case .failed(let error):
+      print(error)
+    default:
+      print("")
+    }
+  })
+}
+```
+As a result of passing in a URL of an image of a celebrity, you should see the corresponding celebrity name printed to the screen along with additional metadata.
```

Comment:
should or will? *audience canned laughter* I kind of hate seeing "should" as a reader. It's a sign of weakness on the part of the writer. We can be precise and say "if a celebrity is detected, you will see their name in the result"

Reply:
okay, i can change this to "well known celebrity, you will see" the corresponding celebrity name